politeiawww/cmsaddresswatcher.go

```diff
@@ -30,7 +30,7 @@ const (
 )
 
 func (p *politeiawww) addWatchAddress(address string) {
-	err := p.wsDcrdata.subToAddr(address)
+	err := p.wsDcrdata.subscribe(addrSubPrefix + address)
```

Comment:
Don't do this.  Use the `wsDcrdata.subscribe()` method in the `subToAddr` method instead of the client `Subscribe()` method.